incubator/catalog-apicurio/src/main/java/io/aklivity/zilla/runtime/catalog/apicurio/internal/ApicurioEventContext.java

```diff
@@ -14,23 +14,28 @@
  */
 package io.aklivity.zilla.runtime.catalog.apicurio.internal;
 
+import static io.aklivity.zilla.runtime.catalog.apicurio.internal.types.event.ApicurioEventType.REMOTE_ACCESS_REJECTED;
+
 import java.net.http.HttpRequest;
 import java.nio.ByteBuffer;
 import java.time.Clock;
 
-import org.agrona.MutableDirectBuffer;
+import org.agrona.concurrent.AtomicBuffer;
```

Comment:
Please use `MutableDirectBuffer` to declare variables if atomic-specific methods are not used.